integreat_cms/deepl_api/deepl_api_client.py

```diff
@@ -147,13 +150,18 @@ def translate_queryset(
                     if hasattr(source_translation, attr) and getattr(
                         source_translation, attr
                     ):
-                        # data has to be unescaped for DeepL to recognize umlauts
-                        data[attr] = self.translator.translate_text(
-                            unescape(getattr(source_translation, attr)),
-                            source_lang=source_language.slug,
-                            target_lang=target_language_key,
-                            tag_handling="html",
-                        )
+                        try:
+                            # data has to be unescaped for DeepL to recognize Umlaute
+                            data[attr] = self.translator.translate_text(
+                                unescape(getattr(source_translation, attr)),
+                                source_lang=source_language.slug,
+                                target_lang=target_language_key,
+                                tag_handling="html",
+                            )
+                        except DeepLException as e:
+                            deepl_api_error = True
+                            logger.error(e)
+                            break
```

Comment:
```suggestion
                        except DeepLException as e:
                            messages.error(
                                self.request,
                                _(
                                    "A problem with DeepL API has occurred. Please contact an administrator."
                                ),
                            )
                            logger.error(e)
                            return
```

Assuming that we don't need 2 error messages here?
![image](https://github.com/digitalfabrik/integreat-cms/assets/115008338/d3598f84-14f3-4785-8b22-f55d938519f8)
The top message occurs because the condition `if content_translation_form.is_valid()` is not met. It also adds the following message to the log:
> Mar 26 10:01:35 ERROR integreat_cms.deepl_api.deepl_api_client - Automatic translation for <Page (id: 4, parent: 1, region: augsburg, slug: interesting-facts-about-augsburg)> could not be created because of {'title': ['This field is required.'], 'slug': ['Cannot generate slug from title.']}
Seems irrelevant to me when there's a deepl error.